### PR TITLE
Rs 2022 08 02

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-08-03-215736_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 1]
```diff
@@ -57,47 +58,54 @@ The creaking in my ankles earlier was a sign that my time has come.
...
 -- Page 305, Offset 11680.
 -- ―――失敗した。
-―――I lost.
+―――I've lost.
 }
 [sha:68befa1d381f200e7144dbdcf9b0e6ebb5b8cfd9]{
 -- Page 305, Offset 11681.
 -- こんな単純な動作に対応できない。先ほどとなんら変わらない速度なのに体が重い。動かない。躱せない。これでは、頭を掴まれた瞬間、トマトのように潰される―――
-I can't do anything against such a simple attack. It isn't any faster than last time, but my body feels heavy. I can't move. I can't dodge. With this, the moment he grabs my head, I'll be crushed like a tomato―――
+I can't do anything, even in the face of such a simple attack.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:82]: anything. Even in

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 2]
```diff
@@ -122,28 +130,29 @@ It's no use, my legs still feel heavy...!
...
 -- Page 310, Offset 11693.
 -- 対して。
-On the other hand.
+As for my opponent―――
 }
 [sha:56fa07e130d3e7b10f3ee43f78f4df0e346d9e36]{
 -- Page 310, Offset 11694.
 -- 視界には、彼方に飛ばされながらも平然と立ち上がり、右腕をかざす吸血鬼の姿があった。
-Right there, in my field of view: even after having been blown away, the vampire gets up, holding his right arm over his head.
+In my field of view I can still see the figure of that vampire,
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:153]: view, I can

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 3]
```diff
@@ -238,8 +249,8 @@ It wasn't on my mind earlier, but that voice―――
 [sha:a6a72c1fe5bff485e676f0d2ae2acdf2d83c2036]{
 -- Page 317, Offset 11713.
 -- 「……はあ。できれば最後の時まで秘密にしておきたかったんですけど」
-// ciel my beloved
-"...Haa, if possible, I would've wanted to keep it a secret until the very end."
+// Ciel my beloved
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:252]: image

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 4]
```diff
@@ -279,72 +292,75 @@ Did you hear about Executors from that blonde True Ancestor?"
...
 -- Page 323, Offset 11727.
 -- 「聞いているようですね。話が早くて助かります。
-"It seems she told you. Good, you're quick on the uptake.
+"It seems you have. Good, I don't need to explain then.
 }
 [sha:42588aacd9e6ad2cbaaecdcba2c6f3643ea610d3]{
 -- Page 323, Offset 11728.
 -- ……そういう事ですから、遠野くんはそのまま一歩も動かないように。後の処理は、先生に代わって私がします」
-...That's how it is, so Tohno-kun, please don't move an inch. Instead of Sensei, I'll take care of the rest."
+Well... that's the way it is, so please just stay put for the minute Tohno-kun.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:331]: so please, just

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 5]
```diff
@@ -410,47 +436,52 @@ Even I can see that's she's trying to stir him up.
 [sha:703956460ea11c6e104f855770882c0f3ca5e9a8]{
 -- Page 331, Offset 11747.
 -- 「“……その通りだ、代行者。
-"...Indeed, Executor.
+"It is as you say, Executor...
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:439]: Petrikow was saying that for these pauses, we should add ellipses. can't check ig right now though

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 6]
```diff
@@ -410,47 +436,52 @@ Even I can see that's she's trying to stir him up.
...
 }
 [sha:85741e9fed0218154c8793146f776aeb5b62a3d4]{
 -- Page 331, Offset 11748.
 -- もとより、おれの国に食い物はない。城の倉庫はとうにカラだ。人間どもも一世紀前に逃げ去った。貴様ら狗ですら、おれの国には近寄らん。
-To begin with, food was always scarce in my country. The castle's storehouse has long since been emptied. The humans too, fled a century ago. Even you, dogs, won't come near my land.
+Food has always been scarce in my country.
+ My castle's storehouse has long stood empty.
+ Even the humans fled a century ago.
+ Not even dogs such as yourselves stray onto my lands.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:447]: into my lands

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 7]
```diff
@@ -410,47 +436,52 @@ Even I can see that's she's trying to stir him up.
...
 -- Page 332, Offset 11751.
 -- ヤツは挑発には乗らなかった。そればかりか人間とまともに会話をしている……！
-He wasn't responding to her provocation. Not only that, he was having a proper conversation with a human...!
+Not only did he not rise to her provocation, but he's actually carrying on a normal conversation with a human...!
 }
 [sha:da4b933f75097b765cfe88242a903fd6c3b6d88c]{
 -- Page 333, Offset 11752.
 -- ホテルの時とは何が違うのか。
-Is this different from the time at the hotel?
+What makes this time different to the hotel?
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:469]: from the hotel

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 8]
```diff
@@ -460,134 +491,147 @@ To a man of the countryside, poverty is a virtue, isn't it?"
...
 -- Page 338, Offset 11767.
 -- 「“……そうか。難儀な女だ。
-"...I see. What a difficult woman.
+"I see... What a difficult woman.
 }
 [sha:29518cf65c0ab7f1e76e98d72ff65e3de11cdf46]{
 -- Page 338, Offset 11768.
 -- ではその<師|おんな>の後を追え。貴様と交わす<過去|ことば>はない”」
-Then, follow in the steps of that <woman|t e a c h e r >. I have no more words for you."
+// TLing 師 as teacher here seems misleading, since I assume Vlov doesn't
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:555]: definitely right

though now, no matter which interpretation you choose, it's definitely lost. Not saying ruby is necessary but maybe change "woman" -> "master"?

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 9]
```diff
@@ -460,134 +491,147 @@ To a man of the countryside, poverty is a virtue, isn't it?"
...
 -- Page 340, Offset 11774.
 -- ヤツは敵の不覚を<嘲|わ><笑|ら>うように、
-As if mocking his enemy's negligence,
+As if mocking his enemy's blunder―――
 }
 [sha:e7b98f2183e64bfd7ddbf8c952c6c241adab8ec5]{
 -- Page 340, Offset 11775.
 -- 「“―――それは、一体どの程度だ？”」
-"―――And what level exactly is that?"
+"And just what caliber is that, exactly?"
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:596]: i'm guessing this pause should remain too

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 10]
```diff
@@ -657,32 +706,37 @@ Dumbfounded, I just nod.
...
 -- Page 350, Offset 11797.
 -- こんな地獄の渦中にいながら、あの夜、俺を救ってくれた先輩のままだった。
-In the midst of this hell, she's just like the Senpai that helped me on that night.
+Even in the midst of this hell, she's still the same person that saved me that night.
 }
 [sha:502e11b093e84b8c2b7f5d8931996aadae2be7f7]{
 -- Page 351, Offset 11798.
 -- それが嬉しい反面、不安になる。
-This made me happy, but also uneasy.
+I'm glad, but at the same time, it makes me little uneasy.
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:719]: a little uneasy

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt [hunk 11]
```diff
@@ -657,32 +706,37 @@ Dumbfounded, I just nod.
...
-If %{i}this%{/i} is routine work for her, then what of the person I've known until now―――
+If %{i}this%{/i} is routine work for her,
+ then just how much is there that I still don't I know about this person―――
 }
 [sha:dce46b3a341bbdb4c750052aab934cf4c5c9b50d]{
 -- Page 352, Offset 11800.
 -- 「そろそろですね。……並の魔術師なら一撃で底をつく火力を２分強持続……只の死徒とは思えない呪力ですが、それもここまででしょう。これ以上はさすがに―――」
-"This should be it... Maintaining this much firepower for more than two minutes, when the average mage would be exhausted after a single hit... I wouldn't have expected such a powerful curse from a common Dead Apostle. But this is the end. Any more shouldn't be―――"
+"Shouldn't be long now...
+ The average mage would be exhausted after loosing a single volley of this intensity,
```
Comment (Avuxo) [inline script/Ciel/Day 5/05_01_CIEL05_2_2.txt:731]: losing

**Review by Avuxo — APPROVED** [review]
otsu

File: script/Ciel/Day 5/05_01_CIEL05_2_2.txt (hunk 11 above)
Comment (rschlaikjer) [reply script/Ciel/Day 5/05_01_CIEL05_2_2.txt:731]: Pretty sure I mean loosing here - letting loose a volley

Comment (Avuxo) [reply script/Ciel/Day 5/05_01_CIEL05_2_2.txt:731]: you're right, my mind kinda wandered off after i saw "exhausted after loosing"

i'd change it to "loosing off"

**Pull request merged**